x-pack/plugins/actions/server/routes/connector/execute/transforms/transform_connector_response/v1.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ConnectorExecuteResponseV1 } from '../../../../../../common/routes/connector/response';
+import { ActionTypeExecutorResult } from '../../../../../types';
+
+export const transformExecuteConnectorResponse = ({
+  actionId,
+  retry,
+  serviceMessage,
+  ...res
+}: ActionTypeExecutorResult<unknown>): ConnectorExecuteResponseV1 => ({
+  ...res,
+  connector_id: actionId,
+  ...(retry && typeof retry === 'object' ? { retry: retry.toISOString() } : { retry }),
```

Comment:
is there a way to specifically test for `Date` and not just object?

Reply:
Yes, my bad! Resolved in this commit [9e6c4bb](https://github.com/elastic/kibana/pull/194481/commits/9e6c4bb1deeac1c8e939a719e03060436e719cd9)